### ISSUE TITLE
Bundled .dmg contains reference to unbundled library on Mac OSX

### ISSUE THREAD

**Issue by sofian** [mapmapteam/mapmap#233]
```
Dyld Error Message:
  Library not loaded: /Library/Frameworks/GStreamer.framework/Versions/1.0/lib/libgstapp-1.0.0.dylib
  Referenced from: /Volumes/MapMap-0.4.0-1/MapMap.app/Contents/Frameworks/GStreamer.framework/Versions/1.0/lib/GStreamer
  Reason: Incompatible library version: GStreamer requires version 601.0.0 or later, but libgstapp-1.0.0.dylib provides version 205.0.0
```

**Comment by maxdee** [comment]
This is the error I get on the Mac mini with OSX 10.10.3 and MapMap-0.4.0-1.
I attach the log.
[osx_10_10_mapmap_bug.txt](https://github.com/mapmapteam/mapmap/files/190154/osx_10_10_mapmap_bug.txt)

**Comment by sofian** [comment]
Yes, seems to be the same error. The problem being that the libgstapp-1.0.0.dylib should be referred to relative to the bundle rather than to the absolute path /Library/Frameworks/GStreamer.framework/Versions/1.0. I'll fix it as soon as I can get my hands on a Mac...

**Comment by sofian** [comment]
This script might be useful: https://github.com/EricssonResearch/cerbero/blob/master/cerbero/tools/osxrelocator.py

**Comment by aalex** [comment]
We do have a laptop with OS X 10.11 at PDS! I am there right now, and we would like to run MapMap on it.
